### DISCUSSION TITLE
Option to use regular ZeroTier (LTE) connection instead if WiFi radio

### DISCUSSION THREAD

**Discussion by Sergey-Afonin** [svpcom/wfb-ng#135] [Ideas]
I tried getting FPV video stream via LTE. I used the same tools like ZeroTier, raspivid and hello_video. It works fine, but I guess it could be so nice to add to your project such feature - using regular ZeroTier TCP/UDP connection (via LTE as well) - assignment IP address and ports for counterparty if enough for connect. It is good if you can approach me to points of change if changes are not major.

**Comment by svpcom** [comment]
For LTE you don't need wifibroadcast. Use any full-mesh vpn (for example [tinc](https://www.tinc-vpn.org/))
